docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md

```diff
@@ -39,7 +39,73 @@ Make sure that the following modules have been installed:
 
 {% endinfo_block %}
 
-### 2) Set up transfer objects
+### 2) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN | DESCRIPTION | PREREQUISITES | NAMESPACE |
+|-|-|-|-|
+| MerchantProductProductAbstractPostCreatePlugin | Creates a new merchant product abstract entity if `ProductAbstractTransfer.idMerchant` is set. | None | Spryker\Zed\MerchantProduct\Communication\Plugin\Product |
+| PriceProductAbstractPostCreatePlugin | Creates new product price entities by abstract product id and price type if they don't exist. | None | Spryker\Zed\PriceProduct\Communication\Plugin\Product |
+| PriceProductProductAbstractExpanderPlugin | Expands product abstract transfer with prices. | None | Spryker\Zed\PriceProduct\Communication\Plugin\Product |
+| ImageSetProductAbstractPostCreatePlugin | Persists all provided image sets to database for the given abstract product. | None | Spryker\Zed\ProductImage\Communication\Plugin\Product |
+| ProductImageProductAbstractExpanderPlugin | Expands product abstract transfer with product images. | None | Spryker\Zed\ProductImage\Communication\Plugin\Product |
+| TaxSetProductAbstractExpanderPlugin | Finds tax set in database by `ProductAbstractTransfer.idProductAbstract` and sets ProductAbstractTransfer.idTaxSet transfer property. | None | Spryker\Zed\TaxProductConnector\Communication\Plugin\Product |
+| TaxSetProductAbstractPostCreatePlugin | Saves tax set id to product abstract table. | None | Spryker\Zed\TaxProductConnector\Communication\Plugin\Product |
```

Comment:
Not needed here, this is part of Product feature integration. So plugins adjustments will be moved to `Merchant product feature` IG with `MerchantProductProductAbstractPostCreatePlugin` only